src/containers/IdeLoader.tsx

```diff
@@ -119,8 +121,41 @@ class IdeLoaderContainer extends React.PureComponent<Props, State> {
     }
   }
 
+  private async handleMessage(event: MessageEvent): Promise<void> {
+    if (!event || typeof event.data !== 'string') {
+      return;
```

Comment:
When it's going to happen? Would be it a good idea to at least log console warning?

Reply:
The data sent by the message emitter can be any data type.
https://developer.mozilla.org/en-US/docs/Web/API/MessageEvent/data

It is always possible. We do not need any warnings in this case.

But the additional check ```!event ``` is not necessary. I have removed it.